dex/networks/ltc/params.go

```diff
@@ -18,34 +18,54 @@ var (
 		},
 	}
 	// MainNetParams are the clone parameters for mainnet.
-	MainNetParams = btc.ReadCloneParams(&btc.CloneParams{
-		PubKeyHashAddrID: 0x30,
-		ScriptHashAddrID: 0x32,
-		Bech32HRPSegwit:  "ltc",
-		CoinbaseMaturity: 100,
-		Net:              0xdbb6c0fb,
-	})
+	MainNetParams = &chaincfg.Params{
+		Name:                    "mainnet",
+		Net:                     wire.BitcoinNet(0xdbb6c0fb),
+		CoinbaseMaturity:        100,
+		Bech32HRPSegwit:         "ltc",
+		PubKeyHashAddrID:        0x30,                            // starts with L
+		ScriptHashAddrID:        0x32,                            // starts with M
+		PrivateKeyID:            0xB0,                            // starts with 6 (uncompressed) or T (compressed)
+		WitnessPubKeyHashAddrID: 0x06,                            // starts with p2
+		WitnessScriptHashAddrID: 0x0A,                            // starts with 7Xh
```

Comment:
For example, these two appear to be unused anywhere in the btcd, btcutil, or btcwallet. But someone trying to find these parameters from the C++ code of a non-ported project might waste a lot of time looking for them. They don't appear to exist there, from what I can tell.

Reply:
To note, these lines were added after @davecgh had relinquished control of the repo.

https://github.com/btcsuite/btcd/blame/074266215c26f8c4731330abb515f92b905f8dfa/chaincfg/params.go#L261